packages/sitecore-jss-nextjs/src/components/NextImage.tsx

```diff
@@ -0,0 +1,123 @@
+import { mediaApi } from '@sitecore-jss/sitecore-jss/media';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+  getEEMarkup,
+  ImageProps,
+  ImageField,
+  ImageFieldValue,
+} from '@sitecore-jss/sitecore-jss-react';
+import Image, {
+  ImageLoader,
+  ImageLoaderProps,
+  ImageProps as NextImageProperties,
+} from 'next/image';
+
+type NextImageProps = Pick<
+  ImageProps,
+  'editable' | 'imageParams' | 'field' | 'mediaUrlPrefix' | 'srcSet'
+> &
+  Partial<NextImageProperties>;
+
+export type ImageConfigComplete = {
+  path: string;
+};
+
+export type ImageConfig = Partial<ImageConfigComplete>;
+
+const { path: configPath } = (process.env.__NEXT_IMAGE_OPTS as any) as ImageConfigComplete;
+
+export const loader: ImageLoader = ({ src, width }: ImageLoaderProps): string => {
+  try {
+    const url = new URL(`${configPath}${src}`);
```

Comment:
We should check if `src` is an absolute URL already. i.e. they have `<IncludeServerUrlInMediaUrls>true</IncludeServerUrlInMediaUrls>` (shouldn't) OR they are deployed on Delivery Edge (expected). In that case, don't prepend `configPath`.